From 63e8adad87ade0d13777e6e48808e856168a8fe8 Mon Sep 17 00:00:00 2001
From: QuonXF <129203880+QuonXF@users.noreply.github.com>
Date: Sat, 16 Dec 2023 21:58:12 +0400
Subject: [PATCH] Add Bosto BT-12HD series to hwdb

(cherry picked from commit 6902da549bfd5313df0802f18fab74d4cc126427)

Related: RHEL-5950
---
 hwdb.d/60-evdev.hwdb | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/hwdb.d/60-evdev.hwdb b/hwdb.d/60-evdev.hwdb
index a9ce4e2862..9a90420b5f 100644
--- a/hwdb.d/60-evdev.hwdb
+++ b/hwdb.d/60-evdev.hwdb
@@ -237,6 +237,15 @@ evdev:name:SYNA3602:00 0911:5288 Touchpad:dmi:*svnBANGHO:pnCLOUDPRO:*
  EVDEV_ABS_35=52:1747:17
  EVDEV_ABS_36=45:954:14
 
+#########################################
+# Bosto
+#########################################
+
+# Bosto BT-12HD series
+evdev:input:b0003v0ED1p7821*
+ EVDEV_ABS_00=::2271
+ EVDEV_ABS_01=::5080
+
 #########################################
 # Dell
 #########################################
